lib/mutant/color.rb

```diff
@@ -24,7 +24,7 @@ def format(text)
         text
       end
 
-    private
+    private # rubocop:disable AccessModifierIndentation
```

Comment:
I think I prefer the original code actually. I would keep the existing formatting if it were up to me.